File: src/main/resources/scripts/HelloWorld.js
/*
* HelloWorld.js  -  displays the number of actually open layers
*/

var JOptionPane = javax.swing.JOptionPane;
var MainApplication = org.openstreetmap.josm.gui.MainApplication;

var numLayers = MainApplication.getLayerManager().getLayers().size();
JOptionPane.showMessageDialog(
    MainApplication.getMainFrame(),
    "[JavaScript] Hello World! You have " + numLayers + " layer(s)."
);

File: src/main/resources/scripts/HelloWorld.py
#
# HelloWorld.py  - displays the number of actually open layers
#
from javax.swing import JOptionPane
from org.openstreetmap.josm.gui import MainApplication

num_layers = MainApplication.getLayerManager().getLayers().size()
JOptionPane.showMessageDialog(
    MainApplication.getMainFrame(),
    "[Python] Hello World! You have %s layer(s)." % num_layers)
